bitnami/minio/templates/NOTES.txt

```diff
@@ -4,10 +4,18 @@ MinIO can be accessed via port {{ .Values.service.port }} on the following DNS n
 
    {{ include "minio.fullname" . }}.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}
 
+{{- if and .Values.credentialsFile.enabled (not .Values.credentialsFile.useSecretValues) }}
```

Comment:
Looking at this, I think we could avoid the major version bump if we allow setting either Values.credentialsFile.enabled or useCredentialsFile. This would make the solution backwards compatible. Maybe we could throw a warning if useCredentialsFile is set. Thoughts?

Reply:
Yes, I agree. If there is a way to avoid the breaking change and the major version bump it would be nice to implement it. Do you know any parameter in another Helm Chart using the proposed logic? In this way, Sleepy-GH can use it as an example to perform the suggested changes.

Reply:
I'm waiting for an answer on this. I can try and provide my own backwards compatible fix if that's easier?

Reply:
Hi! In thanos we did something similar. 

https://github.com/bitnami/charts/pull/4307/files#diff-f0d8330197d82de54920f75eac2c5cba46aa5d6918b5ea4fda8d7eb5a82ca3b9R168-R172

The idea would be use an if statement and check whether one value is set or the other. It could be done in a helper funciton.